docker/Dockerfile

```diff
@@ -19,4 +19,6 @@ LABEL description="MOOSEFS CSI plugin"
 
 COPY --from=build-env /moosefs-csi-plugin /bin/moosefs-csi-plugin
 
+RUN apk add moosefs --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing/
```

Comment:
The `FROM` defaults will have the moosefs built for the moosefs version in that tag.  Pulling from edge package repository means this build would no longer be reproducible.  There are a lot of moosefs clusters that are not running the latest version.

Reply:
> The `FROM` defaults will have the moosefs built for the moosefs version in that tag.

@Kunde21 I couldn't find the source for the `kunde21/moosefs` image, but it doesn't seem to be working as expected. Some of the `mfs` executables it installs under `/usr/bin` don't work correctly:
```
/ # /usr/bin/mfscli
env: can't execute 'python2': No such file or directory
```
And `/sbin/mount.moosefs` is missing moosefs so `mount -t moosefs` doesn't work. Perhaps you're running a local image that doesn't have these issues?

> Pulling from edge package repository means this build would no longer be reproducible. There are a lot of moosefs clusters that are not running the latest version.

Good point, my goal here was just to get something working. I'm all for hermetic builds. It would make more sense to build from source in this Dockerfile, although I suppose that's probably the intent behind the `kunde21/moosefs` image. Can we just include those build steps here?